server/src/main/java/com/objectcomputing/checkins/services/team/TeamBulkLoadException.java

```diff
@@ -0,0 +1,14 @@
+package com.objectcomputing.checkins.services.team;
+
+import java.util.List;
+
+public class TeamBulkLoadException extends RuntimeException {
```

Comment:
We may not need this file since we removed bulk loading and looks like it is not being called anymore.

Reply:
done